### PR TITLE
Menyeart tables3

### PR DESCRIPTION
**Pull request by menyeart**
____ Wrote Tests ____ Implemented ____ Reviewed

-Created a migration to change doctors table column 'type' to 'category'. Necessary as type appears to be a graphql reserved word.
-Created a migration to change health events table column 'type' to 'category'
-Updated Doctor and Health Event models and respective spec files.
Necessary checkmarks:

[x] All Tests are Passing

[x] The code will run locally
Type of change

[] New feature
[] Bug Fix
Implements/Fixes:

description closes #
Check the correct boxes

[x] This broke nothing
[] This broke some stuff
[] This broke everything
Testing Changes

[] No Tests have been changed
[x] Some Tests have been changed
[] All of the Tests have been changed(Please describe what in the world happened)
Checklist:

[x] My code has no unused/commented out code
[x] I have reviewed my code
[] I have commented my code, particularly in hard-to-understand areas
[x] I have fully tested my code
(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

Co-Authored by:

### CODE REVIEW

File: db/migrate/20230516003709_rename_type_to_category.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class RenameTypeToCategory < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :doctors, :type, :category
```
Comment (KaraJoHo) [inline db/migrate/20230516003709_rename_type_to_category.rb:3]: Hopefully GraphQL will be nice to us about category

**Review by KaraJoHo — CHANGES_REQUESTED** [review]
Looking good thank you! A tiny favor...the doctors factorybot may need to be changed to category as well. If you can make that change, that would be awesome.

File: spec/factories/doctors.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
     name {Faker::Name.name}
     address {Faker::Address.full_address}
     phone {Faker::PhoneNumber.phone_number}
-    type {Faker::Lorem.word} #Faker::Job.field (but it's not medical specific)
+    category {Faker::Lorem.word} #Faker::Job.field (but it's not medical specific)
```
Comment (KaraJoHo) [inline spec/factories/doctors.rb:6]: Thank you!

**Review by KaraJoHo — APPROVED** [review]

**Review by GreenGogh47 — APPROVED** [review]
Thanks for modifying that attribute name!

**Pull request merged**